### PR TITLE
Cleaning up Build

### PR DESCRIPTION
**Pull request by shrunyan**
I was confused on where I should place my work in the project so I've made a few changes to make it clear where certain files belong.

It's currently setup to serve a `index.html` file that exists in `public`. Public is setup to be ignored and shouldn't exist in the repo. So I've moved the `index.html` to `app` and setup a blank `index.js` file so it's clear where to begin adding custom logic.
- removed `.gitkeep` file
- changed npm scripts to use local binaries to avoid setup issues

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ node_modules
 .jshintrc
 public
 .idea
+.gitkeep
```
Comment (hbrysiewicz) [inline .gitignore:5]: .gitkeep should not be in the .gitignore

File: gulpfile.babel.js [hunk 2]
```diff
@@ -19,8 +20,15 @@ function serve() {
   gulp.watch(['app/**/*.css'], ['styles'])
 }
 
+function html() {
```
Comment (hbrysiewicz) [inline gulpfile.babel.js:23]: should be a task in task dir like the others

File: .gitignore (hunk 1 above)
Comment (shrunyan) [inline .gitignore:5]: Okay I'll remove it. I thought it was a program config file.

File: gulpfile.babel.js (hunk 2 above)
Comment (shrunyan) [inline gulpfile.babel.js:23]: :+1: will do

**Pull request merged**